Gaijinizer patch/Map529.txt

```diff
@@ -3,685 +3,881 @@
-> CONTEXT: Map529/events/8/pages/0/13/Dialogue/enemy2 < UNTRANSLATED
-
+> CONTEXT: Map529/events/8/pages/0/13/Dialogue/enemy2
+This space is far from an ideal world, with
+illusions that even I could pass through if
+I were made aware of my role.#DLCheckLater
 > END STRING
 
 > BEGIN STRING
 こ、これは……
-> CONTEXT: Map529/events/8/pages/0/17/Dialogue/kazuya < UNTRANSLATED
-
+> CONTEXT: Map529/events/8/pages/0/17/Dialogue/kazuya
+W-What is this?#Ron
 > END STRING
 
 > BEGIN STRING
 アンタが見てたのはこいつらが創り出した
 『新しい世界』でもなんでもなかったのよ。
 
 \\C[2]世界の残骸をより合わせただけの、ただの幻覚。
-> CONTEXT: Map529/events/8/pages/0/19/Dialogue/enemy2 < UNTRANSLATED
-
+> CONTEXT: Map529/events/8/pages/0/19/Dialogue/enemy2
+What you were seeing was not the "new world"
+created by this thing, \\C[2]but simply an illusion
+made from the combination of the ruins of the
+world.#Ron
 > END STRING
 
 > BEGIN STRING
 だいたい、一部でもそんな空間が出来てしまってたら
 \\C[10]オールドワンが顕現しない方がおかしいでしょ？\\C[0]
 
 まだまだ、そんな芸当が出来るほど世界は喰われてないわ。
-> CONTEXT: Map529/events/8/pages/0/24/Dialogue/enemy2 < UNTRANSLATED
+> CONTEXT: Map529/events/8/pages/0/24/Dialogue/enemy2
+I mean, it'd be strange \\C[10]for the Old One not to
+manifest \\C[0]if this thing were capable of creating
+even a small part of a space like that.
 
+It hasn't devoured enough of the world to pull
+off a trick like that.#Ron
 > END STRING
 
 > BEGIN STRING
 こいつらは、アンタに倒されそうになったから
 慌ててこういう空間を用意しただけ。
 
 ご丁寧に、洗脳の準備までしておいてね
-> CONTEXT: Map529/events/8/pages/0/29/Dialogue/enemy2 < UNTRANSLATED
-
+> CONTEXT: Map529/events/8/pages/0/29/Dialogue/enemy2
+These two just created this space in a panic
+because they were about to lose. Here they could
+take their time to brainwash you.#Ron
 > END STRING
 
 > BEGIN STRING
 お兄ちゃん……
 我……私たちと一緒に……
 
 幸せで……平和な………
-> CONTEXT: Map529/events/8/pages/0/35/Dialogue/enemy30 < UNTRANSLATED
+> CONTEXT: Map529/events/8/pages/0/35/Dialogue/enemy30
+Onii-chan...
 
+Be with me... with us...
+Happy... peaceful...#Ron
 > END STRING
 
 > BEGIN STRING
 \\N[1]にぃ……だましてたことはあやまるから……
 
 でも、もっと世界を食べれば、本当に創れるから…！！
 だから……！
-> CONTEXT: Map529/events/8/pages/0/40/Dialogue/enemy30 < UNTRANSLATED
+> CONTEXT: Map529/events/8/pages/0/40/Dialogue/enemy30
+I'll apologize for tricking you, \\N[1]-nii.
 
+But I really will be able to make it if I eat up
+more worlds!! So please...!#Ron
 > END STRING
 
 > BEGIN STRING
 ………一番最初に言ったはずだ。
 
 僕は、\\C[2]お前を受け入れるわけにはいかない。\\C[0]
 例えどんな感情を、こちらに向けられていたとしても……
-> CONTEXT: Map529/events/8/pages/0/46/Dialogue/kazuya < UNTRANSLATED
-
+> CONTEXT: Map529/events/8/pages/0/46/Dialogue/kazuya
+I told you before... \\C[2]I can't possibly accept
+you\\C[0], no matter what kind of feelings you have
+for me...#Ron
 > END STRING
 
 > BEGIN STRING
 僕の世界は\\C[4]あの世界\\C[0]だけだ！
 
 だから……返してもらう！
 僕の『幸せ』たちを！
-> CONTEXT: Map529/events/8/pages/0/51/Dialogue/kazuya < UNTRANSLATED
+> CONTEXT: Map529/events/8/pages/0/51/Dialogue/kazuya
+\\C[4]That world \\C[0]is the only world for me!
 
+So give it back!
+Give back our happiness!#Ron
 > END STRING
```

Comment:
"That world" -> "My world"?

Also possibly "my happiness", but we could probably phrase that better too.